logIn/views.py

```diff
@@ -11,6 +11,8 @@ def login_user(request, user_type):
     verif_error = False
     valid_error = False
     login_error = False
+    if request.session.get("is_login", None):
```

Comment:
Django by default provides - request.user.is_authenticated that can be used to check whether the user has logged in or not. So we don't need to store an extra field like is_login to keep track of that.
What you have done would work but it just isn't necessary to store this piece of info and you can check using this - request.user.is_authenticated directly.